circuit_maintenance_parser/parsers/gtt.py

```diff
@@ -33,21 +33,39 @@ def parse_tables(self, tables, data):
                     if groups:
                         data["maintenance_id"] = groups.groups()[0]
                         status = groups.groups()[1]
-                        if status == "Reminder":
+                        if status == "New":
+                            data["status"] = Status["TENTATIVE"]
```

Comment:
I would say that "New" would mean "CONFIRMED"?

Reply:
Yeah, I was on the fence about this one. I can change it.